lib/redis.js

```diff
@@ -131,8 +131,10 @@ module.exports = CoreObject.extend({
   _deploySuccessMessage: function(revisionKey) {
     var success       = green('\nUpload successful!\n\n');
     var uploadMessage = white('Uploaded revision: ')+green(revisionKey);
+    var exampleCommand = white('To activate, execute: \n\n')+
+      green('ember deploy:activate -r ' + revisionKey + '-e ' + this.config.environment);
```

Comment:
Unfortunately you don't have access to the environment via the config object. this will be possible in the upcoming versions of ember-deploy with the deployment-pipeline and the new hooks-architecture.